cmd/buildah/images.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	imagesFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "quiet",
+			Usage: "omit column headings",
+		},
+		cli.BoolFlag{
+			Name:  "qq",
```

Comment:
At first I thought just --q would be good, but then remembered docker images uses --quiet or -q to only show the ids.  Can I vote to change the flags to --omit for the column headings and --quiet for the image ids only?